hedyorg/hedy
# [FIX] Added brackets around keywords in adventures in de

issue #3537

**Description**

Added brackets to the keywords.

**Addresses #3537 for German**

**How to test**

Users can check that they can now see that the keywords that are surrounded by brackets are now translated into the appropriate language. 

**Checklist**
Done? Check if you have it all in place using this list:*
  

If you're unsure about any of these, don't hesitate to ask. We're here to help!
